libindy/src/services/anoncreds/verifier.rs

```diff
@@ -625,6 +625,19 @@ mod tests {
+        let rev_reg_defs: HashMap<RevocationRegistryId, RevocationRegistryDefinitionV1> = HashMap::new();
+        let rev_regs: HashMap<RevocationRegistryId, HashMap<u64, RevocationRegistryV1>> = HashMap::new();
+        let rc = Verifier::new().verify(
+            &proof, &req, &schema, &creds, &rev_reg_defs, &rev_regs
+        ).unwrap();
+    }
```

Comment:
I think it'll be simpler to create a unit-test for `get_non_revoc_interval` function.
And either extend one of existing `anoncreds_demos` integration tests or provide a new one covering that case to avoid hard-coding so big data just for one unit-test.